implementation/pom.xml

```diff
@@ -64,6 +64,14 @@
             <artifactId>jboss-logging</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <!-- Testing -->
+        <dependency>
+             <groupId>junit</groupId>
+             <artifactId>junit</artifactId>
+             <version>4.13</version>
+             <scope>test</scope>
+         </dependency>
```

Comment:
Can't we go to JUnit 5?

Reply:
You should remove the version. It's managed in smallrye-parent. ([but it's still on 4](https://github.com/smallrye/smallrye-parent/blob/6f913e434e55b6841b8a32de1573c2e2d1820fb3/pom.xml#L40)) . @t1 you can log an issue there if you think we should bump to 5. (I have no opinion)

Reply:
I'll remove the version here - and I agree that we can move to 5 in a separate issue.  I'm more familiar with 4, but I'm willing to learn 5.